Fix test 'http/tests/websocket/tests/hybi/upgrade-simple-ws.html'

dpino opened the pull request:
#### f4cdff05b82d04e76d30105e53685b15d642631e
<pre>
Fix test &apos;http/tests/websocket/tests/hybi/upgrade-simple-ws.html&apos;
<a href="https://bugs.webkit.org/show_bug.cgi?id=158480">https://bugs.webkit.org/show_bug.cgi?id=158480</a>

Reviewed by Alexey Proskuryakov.

This test had several errors. First one, it was accessing the property &apos;ws.URL&apos;,
when it should be &apos;ws.url&apos;. The URL attribute of a WebSocket object is
lowercase.

The other error was a very short timeout. Many other tests in the same
folder have a timeout of 3000ms, while this test had a timeout of 300ms.

* LayoutTests/TestExpectations:
* LayoutTests/http/tests/websocket/tests/hybi/upgrade-simple-ws-expected.txt:
* LayoutTests/http/tests/websocket/tests/hybi/upgrade-simple-ws.html:

Canonical link: <a href="https://commits.webkit.org/299057@main">https://commits.webkit.org/299057@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/31757b299801d8a9d61a8cb03c3b0c20662fa5c5

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/117472 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/37142 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/27762 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/123574 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/69463 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/37838 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/45729 "Built successfully") | [❌ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/89151 "Found 1 new test failure: imported/w3c/web-platform-tests/css/css-transforms/transform-box/cssbox-content-box-002.html (failure)") | [❌ 🧪 win-tests](https://ews-build.webkit.org/#/builders/60/builds/43809 "6 failures") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/120424 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/30126 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/105333 "Passed tests") | [❌ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/69660 "Found 40 new API test failures: /TestJSC:/jsc/basic, /WPE/TestWebKitNetworkSession:beforeAll, /WPE/TestNetworkProcessMemoryPressure:beforeAll, /WPE/TestWebKitWebContext:beforeAll, /WPE/TestSSL:beforeAll, /WPE/TestWebKitURIUtilities:beforeAll, /WPE/TestWebKitWebView:beforeAll, /WPEPlatform/TestDisplayDefault:/wpe-platform/Display/default, /WPE/TestFrame:beforeAll, /TestWebCore:GStreamerTest.harnessParseMP4 ... (failure)") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/133/builds/29183 "Passed tests") | [❌ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/23447 "Exiting early after 10 failures. 10 tests run. 1 failures") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/67246 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/99531 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/23629 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/126692 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/44369 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/33360 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/97818 "Passed tests") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/44727 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/101567 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/97612 "Passed tests") | 
| | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/42983 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/20920 "Passed tests") | [✅ 🛠 playstation](https://ews-build.webkit.org/#/builders/134/builds/40721 "Built successfully") | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/18777 "Built successfully and passed tests") | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/44242 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/43699 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/47043 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/45394 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

aproskuryakov reviewed (approved):
This is fine to land, although I'd expect that 3000ms timeout could still be flaky.

dpino committed:
Fix test 'http/tests/websocket/tests/hybi/upgrade-simple-ws.html'
https://bugs.webkit.org/show_bug.cgi?id=158480

Reviewed by Alexey Proskuryakov.

This test had several errors. First one, it was accessing the property 'ws.URL',
when it should be 'ws.url'. The URL attribute of a WebSocket object is
lowercase.

The other error was a very short timeout. Many other tests in the same
folder have a timeout of 3000ms, while this test had a timeout of 300ms.

* LayoutTests/TestExpectations:
* LayoutTests/http/tests/websocket/tests/hybi/upgrade-simple-ws-expected.txt:
* LayoutTests/http/tests/websocket/tests/hybi/upgrade-simple-ws.html:

Canonical link: https://commits.webkit.org/299057@main
